### DISCUSSION TITLE
Login with multiple providers, then link to Discord?

### DISCUSSION THREAD

**Discussion by n1ckoates** [nextauthjs/next-auth#2556] [Help]
I'm wanting users of my Next app to be able to login with multiple providers - Email, Zoom, or Google. Once they're signed in, they would be able to authenticate with Discord. 

Is this possible to do with `next-auth`? I'm wanting to let users into a specific Discord server once they've authenticated with any of the above providers.

Thanks for any help!

**Comment by danhab99** [comment]
I found mention of what we want to do [here](https://next-auth.js.org/configuration/events#linkaccount). I'm not 100% sure how to use this though?

**Reply by danhab99** [reply]
There's also [this](https://next-auth.js.org/tutorials/creating-a-database-adapter#example-code), but this isn't helpful either. Docs need to be clarified.
